docsrc/imap/reference/manpages/systemcommands/reconstruct.rst

```diff
@@ -138,33 +138,33 @@ Options
     this deletes corrupt message files for ever - so make sure you've
     exhausted other options first!
 
-.. option:: -o
+.. option:: -o, --ignore-odd-files
 
     Ignore odd files in your mailbox disk directories.  Probably useful
     if you are using some tool which adds additional tracking files.
 
-.. option:: -O
+.. option:: -O, --delete-odd-files
```

Comment:
`--ignore-odd-files` and `--delete-odd-files` are mutually exclusive.  Which of them is the default?

Reply:
No idea, maybe neither.  I'm not trying to rewrite the documentation here, but I'll have a look and maybe tweak it a little.

At a guess, I think the default is probably to complain about the odd files, in which case ignore probably shuts up the complaints, whereas delete removes them.  But that's just a guess.

Reply:
My guess was right: by default, it complains about odd files.  Ignore makes it not complain; delete makes it delete them.  I've tweaked the documentation a little to hopefully make this more clear.